packages/react-aria-components/src/GridList.tsx

```diff
@@ -508,7 +512,9 @@ export interface GridListLoadingSentinelProps extends Omit<LoadMoreSentinelProps
   isLoading?: boolean
 }
 
-export const UNSTABLE_GridListLoadingSentinel = createLeafComponent('loader', function GridListLoadingIndicator<T extends object>(props: GridListLoadingSentinelProps, ref: ForwardedRef<HTMLDivElement>, item: Node<T>) {
```

Comment:
the generic `T` seems to be unused if I've understood the types correctly

Reply:
yeah I think you are right, not sure why I used the generic tbh but this looks correct